pkg/manager/manager.go

```diff
@@ -38,11 +37,10 @@ var mgr Manager
 type Manager struct {
 	ConfigStore             *store.ConfigurationStore
 	ChangeStore             *store.ChangeStore
-	DeviceStore             *topocache.DeviceStore
-	NewDeviceStore          devicetopo.Store
+	DeviceStore             devicetopo.Store
 	NetworkStore            *store.NetworkStore
 	ModelRegistry           *modelregistry.ModelRegistry
-	TopoChannel             chan events.TopoEvent
+	TopoChannel             chan *devicepb.ListResponse
```

Comment:
nice - you're taking it straight from the gRPC - no intermediate TopoEvent needed then

Reply:
yes, one less level of transaction needed. I couple TOPO with CONFIG a bit tighter but it's a tradeoff I'm willing to make for sepmplification's sake